single-editorial.php

```diff
@@ -28,77 +28,88 @@
     if (!empty($slides)) {
       if ($is_recent) {
 ?>
-
+      
       <!-- Product details container -->
-      <div class="col col-s-12 col-m-3 set-content-height">
+      <div class="col col-s-10 offset-s-1 col-m-3 offset-m-0 column justify-between">
```

Comment:
ah thats from before we moved the pagination buttons. when the pagination was in that column and needed to stick to the bottom. i'll take those classes out